### ISSUE TITLE
--fix seems to have no effect

### ISSUE THREAD

**Issue by langdonx** [vuejs/eslint-plugin-vue#457]
**Tell us about your environment**

* **ESLint Version:** v4.5.0
* **eslint-plugin-vue Version:** 4.4.0
* **Node Version:** 6.7.0

**Please show your full configuration:**

```
{
    "extends": "plugin:vue/essential",
    "rules": {
        "comma-dangle": [
            "error",
            {
                "arrays": "always-multiline",
                "objects": "always-multiline",
                "imports": "always-multiline",
                "exports": "always-multiline",
                "functions": "ignore"
            }
        ],
        "quotes": [
            "error",
            "single"
        ]
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
    <div>
    </div>
</template>

<script>
module.exports = {
    data: () => ({
        a: 'true',
        b: "false"
    }),
    methods: {}
};
</script>

<style>

</style>
```

**What did you expect to happen?**

I expected the double quotes to be replaced with single quotes, and a comma to be added on line 4.

**What actually happened? Please include the actual, raw output from ESLint.**

Nothing happened, the warnings were outputted still.

I created a test.js file with the contents of the `script` tag and it seemed to work OK:

![image](https://user-images.githubusercontent.com/441982/38471867-25576176-3b45-11e8-8999-2338305f3ee3.png)

![image](https://user-images.githubusercontent.com/441982/38471869-316338f0-3b45-11e8-8a7f-fd8017eeae48.png)

**Comment by michalsnik** [comment]
Same code and configuration works fine on OSX. I don't have any windows machines to test if that might be the case ;/

**Comment by ota-meshi** [comment]
The fix option seems to be available after eslint 4.7.0.
https://github.com/vuejs/eslint-plugin-vue/blob/master/README.md#grey_exclamation-requirements

**Comment by michalsnik** [comment]
Feel free to reopen if you're still experiencing similar issues @langdonx But make sure you use eslint >= 4.7.0 as @ota-meshi pointed
